atomic_reactor/plugins/pre_yum_proxy.py

```diff
@@ -0,0 +1,162 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from __future__ import absolute_import
+
+import os
+
+from six.moves.urllib.parse import urlsplit, urlunsplit
+from six.moves.configparser import ConfigParser
+
+from atomic_reactor.plugin import PreBuildPlugin
+from atomic_reactor.plugins.pre_reactor_config import get_yum_proxies
+from atomic_reactor.util import df_parser
+
+
+class YumProxyPlugin(PreBuildPlugin):
+    key = 'yum_proxy'
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow):
+        """
+        Plugin initializer
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        """
+        super(YumProxyPlugin, self).__init__(tasker, workflow)
+
+        # TODO: Is this dir accessbile in the build context?
```

Comment:
Yes, not removed until after exit plugins have run.
https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/inner.py#L525